### PR TITLE
AnonClass get own class name test

### PR DESCRIPTION
**Pull request by Ailloviee**
Added a test to check if an anonymous class containing literals(a String
in the test case) of its own class name will return the correct value

Signed-off-by: Charles_Zheng <Juntian.Zheng@ibm.com>

### CODE REVIEW

File: test/functional/Jsr335/src/org/openj9/test/jsr335/defineAnonClass/TestUnsafeDefineAnonClass.java [hunk 1]
```diff
@@ -24,6 +24,9 @@
 import org.testng.annotations.Test;
 import org.testng.Assert;
 import org.testng.AssertJUnit;
+
+import java.io.ByteArrayOutputStream;
```
Comment (tajila) [inline test/functional/Jsr335/src/org/openj9/test/jsr335/defineAnonClass/TestUnsafeDefineAnonClass.java]: remove unused imports

**Comment by tajila** [comment]
@llxia please review

**Comment by llxia** [comment]
other than Tobi's comment, the rest LGTM.
For the future push, please add `[ci skip]` in the commit msg to avoid SDK compilation.

**Comment by Ailloviee** [comment]
@llxia I've addressed the comment

**Comment by llxia** [comment]
jenkins test sanity xlinuxlargeheap jdk10

**Comment by llxia** [comment]
jenkins test sanity xlinuxlargeheap jdk8

**Pull request merged**